README.md

```diff
@@ -270,57 +259,57 @@ We use the [standard](https://github.com/testdouble/standard#how-do-i-run-standa
 To check all the offenses throughout the codebase:
 
 ```bash
-$ bin/rails standard
+$ bundle exec standardrb
 ```
 
-To automatically fix all offenses,
+To fix any offenses that standard can autofix, run
 
 ```bash
 $ bundle exec standardrb --fix
 ```
 
-**Note**: Some files have been temporarily ignored under a `.standard_todo.yml`. As we fix these files, remove them from the `yml` file so that they can be picked up by `standard` again for future offenses. Refer to [usage](https://github.com/testdouble/standard#usage) on how to generate todo files.
-
-**Note**: Some files are permanently ignored under `.standard.yml` and do not require linting.
-
 ### Generating seed data
 
-To generate seed (fake patients) data, execute the following command from the project root
+NOTE: Its highly recommended to tune your local PostgreSQL before generating new seed data, especially large seed data sets. See the docs for that under [Development](#development).
+To generate a full set of seed data, including facilities, users, patients with BPs, etc, run the following:
 
 ```bash
-$ bin/rails db:seed_patients
+bin/rails db:seed
 ```
 
-Need a larger dataset? Try adding the `SEED_TYPE` variable. (It takes longer to run, of course.) This works on all patient seeding commands:
+You can always do a full reset to get back to a working dataset locally - note that reset clears all DBs, recreates them, runs seed, and refreshes matviews.
 
 ```bash
-  SEED_TYPE=medium bin/rails db:seed_patients
-# You also may want an entirely new large dataset, with more facilities and regions. You can do that with a full reset:
-SEED_TYPE=large bin/rails db:reset
-
-Available sizes are `small`, `medium`, and `large`. (Large takes a LONG time to run. You probably don't need it.)
+bin/rails db:reset
+```
 
-To purge the generated patient data, run
+Need a larger dataset? Try adding the `SEED_TYPE` ENV variable. Available sizes are `small`, `medium`, and `large`, and `profiling`. Large and profiling take a long time to run (20 mins to an hour), but they are very helpful for performance testing.
 
 ```bash
-$ bin/rails db:purge_users_data
+SEED_TYPE=medium bin/rails db:reset
+# You also may want an entirely new large dataset, with more facilities and regions, and more patients per facility.
+SEED_TYPE=large bin/rails db:reset
 ```
 
-You can also purge and re-seed by running:
+To purge the generated patient data _only_, run the following. Note that you usually don't want this, and a full `db:reset` is safer in terms of generating a valid data set.
 
 ```bash
-$ bin/rails db:purge_and_reseed
+$ bin/rails db:purge_users_data
```

Comment:
I almost removed this, but I imagine it still is useful for some isolated use cases.